cmake/FindLuaJIT.cmake

```diff
@@ -44,5 +44,6 @@ endif()
 if(LuaJIT_FOUND AND NOT TARGET LuaJIT::LuaJIT)
     add_library(LuaJIT::LuaJIT INTERFACE IMPORTED)
     target_include_directories(LuaJIT::LuaJIT INTERFACE ${LuaJIT_INCLUDE_DIRS})
+    target_link_directories(LuaJIT::LuaJIT INTERFACE ${LuaJIT_LIBRARY_DIRS})
     target_link_libraries(LuaJIT::LuaJIT INTERFACE ${LuaJIT_LIBRARIES})
```

Comment:
Is this line still needed?

Reply:
`LuaJIT_LIBRARIES` just has the `luajit-5.1`, but if that isn't in a known path then it fails to find it.  So I think it needs both.

Reply:
From CMakeCache.txt

```
LuaJIT_LIBRARY_DIRS:INTERNAL=/opt/homebrew/Cellar/luajit/2.1.0-beta3-20230430.1/lib
LuaJIT_LIBRARIES:INTERNAL=luajit-5.1
```

Reply:
Interesting. Thanks for the fix.